scrapy/extensions/feedexport.py

```diff
@@ -146,6 +146,32 @@ def _store_in_thread(self, file):
             key.close()
 
 
+
+class GCSFeedStorage(BlockingFeedStorage):
+
+    project_id = None
+    bucket_name = None
+    blob_name = None
+
+    def __init__(self, uri, project_id):
+        self.project_id = project_id
+        u = urlparse(uri)
```

Comment:
Note that these three lines are similar to `S3FeedStorage`.
I thought about extracting them in a method, but it does seem to hurt right now

Reply:
> but it does seem to hurt right now
It DOES hurt? as in... performance?

Reply:
Sorry @stav, I meant _it doesn't_